apps/web/postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  plugins: {
+    tailwindcss: {},
+    autoprefixer: {},
+  },
+}
```

Comment:
I'm not 100% sure we need this but I followed tailwind's doc to integrate with next.js.